### ISSUE TITLE
Theme switcher

### ISSUE THREAD

**Issue by issamadiem** [bizley/timeclock#30]
Hello, first of all I'd like to congragulate you for your amazing work. I just have a slight problem with the theme switche, it doesn't seem to work for me as the route "profile/theme/dark"  is not found.

**Comment by bizley** [comment]
Thank you for your kind words, sir.
As for the theme switcher - yes, I can see the potential problem where you are not using pretty urls - so the way the url in switcher is generated it kind of assumes pretty urls enabled, and when it's not it might not work. Is it your case? The fix is quite easy, I just never found time to correct it, sorry...

**Comment by issamadiem** [comment]
Yes indeed I have the pretty urls option disabled, I'll just turn it back on then, thank you for your reponse.
